src/handlers/ecommerce/payment-method.ts

```diff
@@ -1,9 +1,82 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable sort-keys */
 import { RequestHandler } from "express";
+import fetch from "node-fetch";
+import { pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/lib/Either";
 import { logger } from "../../logger";
 import { createSuccessGetPaymentMethods } from "../../utils/ecommerce/payment-method";
+import { CreateSessionResponse } from "../../generated/ecommerce_payment_methods/CreateSessionResponse";
+import { ProblemJson } from "../../generated/ecommerce_payment_methods/ProblemJson";
+import { Field } from "../../generated/ecommerce_payment_methods/Field";
 
 export const ecommerceGetPaymentMethods: RequestHandler = async (req, res) => {
   logger.info("[Get payment-methods ecommerce] - Return success case");
   res.status(200).send(createSuccessGetPaymentMethods());
 };
+
+const { X_API_KEY } = process.env;
```

Comment:
Add this variable X_API_KEY into env.example file. If possible let's give it a more meaningful name to define that it is NPG's api key 🙂